test/functional/apps/management/_scripted_fields.js

```diff
@@ -138,6 +135,7 @@ export default function ({ getService, getPageObjects }) {
       it('should see scripted field value in Discover', async function () {
         const fromTime = 'Sep 17, 2015 @ 06:31:44.000';
         const toTime = 'Sep 18, 2015 @ 18:31:44.000';
+        await PageObjects.common.sleep(1000 * 10);
```

Comment:
I think this sleep was just for debugging?  And I think the issue wasn't timing but was perhaps 2 index patterns with the same name?